scripts/sfpshow

```diff
@@ -402,6 +437,54 @@ class SFPShow(object):
 
         return output
 
+    def convert_pm_prefix_to_threshold_prefix(self, pm_prefix):
+        if pm_prefix == 'uncorr_frames':
+            return 'postfecber'
+        return pm_prefix.replace('_', '')
+
+    def convert_interface_sfp_pm_to_cli_output_string(self, state_db, interface_name):
+        sfp_pm_dict = state_db.get_all(
+            self.db.STATE_DB, 'TRANSCEIVER_PM|{}'.format(interface_name))
+        sfp_threshold_dict = state_db.get_all(
+            state_db.STATE_DB, 'TRANSCEIVER_DOM_THRESHOLD|{}'.format(interface_name))
+        table = []
+        indent_num = 4
+        indent = ' ' * indent_num
+        if sfp_pm_dict:
+            output = ZR_PM_APPLICABLE_STR + '\n' + indent
+            for param_name, (unit, prefix) in ZR_PM_INFO_MAP.items():
+                row = [param_name, unit]
+                values = []
+                for suffix in ZR_PM_VALUE_KEY_SUFFIXS:
+                    key = prefix + '_' + suffix
+                    values.append(float(sfp_pm_dict[key]))
```

Comment:
For displaying the min, avg and max for "Pre-FEC BER", can we use 0.2E since BERs will be 3.11e-4.
Clarified this with @qinchuanares. Please feel free to correct me if I am wrong.

Reply:
updated to display as 0.2E 
```
    Parameter        Unit    Min       Avg       Max       Threshold    Threshold    Threshold     Threshold    Threshold    Threshold
                                                           High         High         Crossing      Low          Low          Crossing
                                                           Alarm        Warning      Alert-High    Alarm        Warning      Alert-Low
    ---------------  ------  --------  --------  --------  -----------  -----------  ------------  -----------  -----------  -----------
...
    Pre-FEC BER      N/A     4.58E-04  4.66E-04  5.76E-04  1.25E-02     1.10E-02     0.0           0.0          0.0          0.0
...
```